frontend/model/notifications/templates.js

```diff
@@ -107,49 +126,67 @@ export default ({
     const rootState = sbp('state/vuex/state')
     return {
       avatarUserID: data.memberID,
+      title: rootState[data.groupID]?.settings?.groupName || L('Member added'),
       body: L('The group has a new member. Say hi to {strong_}{name}{_strong}!', {
         name: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${data.memberID}`,
         ...LTags('strong')
       }),
+      plaintextBody: L('The group has a new member. Say hi to {strong_}{name}{_strong}!', {
+        name: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${data.memberID}`
+      }),
```

Comment:
`{strong}` still here

Reply:
Yes, that's intentional to avoid having to translate the strings twice. Note that there's no `LTags('strong')` and that the `L` function replaces these missing references with `''`. In effect, this is eliding the `{strong_}` and `{_strong}`.

Reply:
Ah, makes senes.

> the L function replaces these missing references with ''

I forgot it was able to do that!